cmsplugin_articles_ai/models/articles.py

```diff
@@ -153,6 +153,9 @@ class Article(PublisherModel):
     modified_at = models.DateTimeField(
         _("last modified"), auto_now=True, editable=False,
     )
+    show_lead = models.BooleanField(
+        _("show lead"), default=True, help_text=_("Show lead text."),
```

Comment:
It is true that random fields will bloat the model. I think the best option would be to include also lead paragraph field to the model. Many other article apps have separate fields for main content and lead paragraph. Updated this pull request to include lead_paragraph and show_lead fields.